zipline/pipeline/loaders/blaze/core.py

```diff
@@ -548,13 +580,28 @@ def from_blaze(expr,
         is passed, a ``BoundColumn`` on the dataset that would be constructed
         from passing the parent is returned.
     """
-    deltas = get_deltas(expr, deltas, no_deltas_rule)
-    if deltas is not None:
+    deltas = _get_metadata(
+        'deltas',
+        expr,
+        deltas,
+        no_deltas_rule,
+    )
+    checkpoints = _get_metadata(
+        'checkpoints',
+        expr,
+        checkpoints,
+        no_checkpoints_rule,
+    )
+    if {deltas, checkpoints} != {None}:
```

Comment:
oh, this was not the full check that I wanted, I need to fix this. I think it should be: `if 'auto' in (deltas, checkpoints)`